samples/CustomRouteSample.Web/Controllers/CustomRoute_OrdersControlller.cs

```diff
@@ -3,7 +3,7 @@
 
 using Microsoft.AspNet.Mvc;
 
-namespace CustomRouteWebSite.Controllers
+namespace CustomRouteSample.Web.Controllers
```

Comment:
We have been consistent about this naming pattern in the Samples folder up until another @ryanbrandenburg created the `MvcSandbox` project. I don't recall how the convention got started. But if I removed it here, I'd rename `MvcMinimalSample.Web` (well, no, that's going to be a functional test site soon) and `TagHelperSample.Web`.

@kichalla's "Part 1" PRs would also have to change if I changed things here.